app/graphql/types/git/file_type.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+Types::Git::FileType = GraphQL::ObjectType.define do
```

Comment:
Maybe add a field to find the commit that last changed the file? Same for Directory (see directory_entry_type.rb)